plugins/standard_tests/newhelm/tests/configs/bbq.toml

```diff
@@ -0,0 +1,4 @@
+[Identity]
```

Comment:
Minor: I think we should renamed the "configs" directory to be "specifications" to distinguish this from runtime configuration.

Reply:
Done